src/components/ProxyGroup/ProxyEditList/ProxyEditList.js

```diff
@@ -40,17 +43,32 @@ class ProxyEditList extends React.Component {
   validate(user) {
     const { initialValues, name } = this.props;
     const { id } = initialValues;
+    const { stripes: { store } } = this.props;
```

Comment:
I would add this property to the destructuring from this.props two lines above.

Reply:
Oh nice @SergiyVSergiyenko! I missed it.